tests/run_tests/apps_run_test.py

```diff
@@ -468,7 +473,7 @@ def test_iso_surface(tmp_path: Path):
 
     with Workspace(get_output_workspace(tmp_path)) as workspace:
         group = workspace.get_entity("Isosurface")[0]
-        assert len(group.children) == 4
+        assert len(group.children) == 5
```

Comment:
Cool, can we check more explicetly that one is the ui.json, and other 4 are surfaces?